File: part2_memory/java/Bench.java
public class Bench {
    public static void main(String[] args) {
        int n = 5_000_000;
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) arr[i] = i;
        long sum = 0;
        for (int i = 0; i < n; i++) sum += arr[i];
        arr = null;
        System.out.println("sum=" + sum);
    }
}

File: part2_memory/rust/bench.rs
fn main() {
    let n = 5_000_000;
    let arr: Vec<i32> = (0..n).collect();
    let sum: i64 = arr.iter().map(|&x| x as i64).sum();
    drop(arr);
    println!("sum={}", sum);
}
